people-pro/src/components/equipamentos/CadastroEquipamentos.vue

```diff
@@ -0,0 +1,157 @@
+                    <input class="form-check-input" type="radio" :value="arm" :id="arm" v-model="equipamento.caracteristicas.armazenamento.tipo">
+                </div>
+                <div v-if="v$.equipamento.caracteristicas.armazenamento.tipo.$error" :class="['submitError']">Selecione um tipo de armazenamento</div> 
+
+                <label for="equipamento-os">Sistema Operacional</label>
+                <div v-for="os in details.os.pc" :key="os" class="form-check" id="equipamento-os">
+                    <label :for="os">{{ os }}</label>
+                    <input class="form-check-input" type="radio" :value="os" :id="os" v-model="equipamento.caracteristicas.os">
+                </div>
+                <div v-if="v$.equipamento.caracteristicas.os.$error" :class="['submitError']">Selecione um sistema operacional</div>   
+
+            </div>
+
+            <div v-else-if="equipamento.tipoEquipamento==='Celular'">
+                <label for="equipamento-os">Sistema Operacional</label>
+                <div v-for="os in details.os.mobile" :key="os" class="form-check" id="equipamento-os">
+                    <label :for="os">{{ os }}</label>
+                    <input class="form-check-input" type="radio" :value="os" :id="os" v-model="equipamento.caracteristicas.os">
+                </div>
+                <div v-if="v$.equipamento.caracteristicas.os.$error" :class="['submitError']">Selecione um sistema operacional</div>    
+            </div>
+
+            <button
+                type="submit"
+                class="btn btn-primary"
+                @click.prevent="cadastrarEquipamento"
+              >Cadastrar</button>
+        </form>   
+    </div> 
+</template>
+
+<script>
+import { useVuelidate } from '@vuelidate/core'
+import { required, minValue, requiredIf } from '@vuelidate/validators'
+export default{
+    setup () {
+    return { v$: useVuelidate() }
+    },
+    data() {
+        return {
+            equipamento : {
+                tipoEquipamento:'',
+                marca:'',
+                serial:'',
+                descricao:'',
+                dataDeCompra:'',
+                ultimaAtualizacao:'',
+                caracteristicas: {
+                    ram:'',
+                    armazenamento:{tipo:'', capacidade:0},
+                    os:'',
+                    gpu:''
+                },
+            },
+            details:{
+                tipos:['Celular','Desktop','Notebook'],
+                os: {
+                    pc:["Windows","Linux","macOS"],
+                    mobile:["iOS","Android"]
+                },
+                armazenamento:{
+                tipo:['HD','SSD'],
+                capacidade:0
+                },
+                gpu:''
+            },
+            equipamentosDisponiveis:[]
+        }       
+    },
+    methods: {
+        async cadastrarEquipamento(){
+            const isValid = await this.v$.$validate();
+            if(!isValid)
```

Comment:
Pode substituir a estrutura `if/else` pelo seguinte:
```suggestion
            if(isValid) this.equipamentosDisponiveis.push(this.equipamento);
```